src/plugins/paginate/_base.scss

```diff
@@ -24,6 +24,67 @@
 		border-bottom-right-radius: 4px;
 		border-top-right-radius: 4px;
 	}
+
```

Comment:
As discussed, please convert the newly added lines to SASS. We can discuss further when you believe you are done.

Reply:
Fixed in commit "Made recommended changes".